### PR TITLE
update app config doc

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: doc/sphinx/ref/azure.appconfiguration.rst [hunk 1]
```diff
@@ -19,26 +19,18 @@ azure.appconfiguration.azure\_appconfiguration\_client module
    :undoc-members:
    :show-inheritance:
 
-azure.appconfiguration.azure\_appconfiguration\_requests module
----------------------------------------------------------------
+azure.appconfiguration.azure\_appconfiguration\_credential module
+-----------------------------------------------------------------
```
Comment (bryevdv) [inline doc/sphinx/ref/azure.appconfiguration.rst]: What's up with the `\`? Is that something specific to our sphinx setup? I have never had to do this in other projects where there were module names with underscores.

Comment (xiangyan99) [reply doc/sphinx/ref/azure.appconfiguration.rst]: '\' is used in sphinx to escape '_'. The full name of the module is 
azure.appconfiguration.azure_appconfiguration_requests

Comment (bryevdv) [reply doc/sphinx/ref/azure.appconfiguration.rst]: Right, my confusion is that I've never known underscores to ever need escaping before :)  e.g. 

https://raw.githubusercontent.com/bokeh/bokeh/master/sphinx/source/docs/reference/tile_providers.rst

(with no escapes for underscores) generates

http://docs.bokeh.org/en/latest/docs/reference/tile_providers.html

Comment (xiangyan99) [reply doc/sphinx/ref/azure.appconfiguration.rst]: This is generated by the sphinx tool. :)

**Pull request merged**